docs/pages/components/side-navigation.md

```diff
@@ -292,31 +297,32 @@ In compact mode the dimensions of the controls are reduced, allowing more inform
 The lists in both sections (Main and Utility) should have the `fd-nested-list--compact` modifier class.
 
 {% capture default %}
-<nav class="fd-side-nav">
-    <div class="fd-side-nav__main-navigation">
-        <ul class="fd-nested-list fd-nested-list--compact">
-            <li class="fd-nested-list__group-header">
-                Group Header 1
-            </li>
+<div class="fd-side-nav">
+    <a class="fd-side-nav__skip-link" href="#content">Skip navigation</a>
+    <div class="fd-side-nav__group-header" id="EX500H1">
+        Group Header 1
+    </div>
+    <nav class="fd-side-nav__main-navigation">
+        <ul class="fd-nested-list fd-nested-list--compact" aria-labelledby="EX500H1">
             <li class="fd-nested-list__item">
                 <a class="fd-nested-list__link" href="#">
-                    <span class="fd-nested-list__icon sap-icon--home"></span>
+                    <span aria-hidden="true" class="fd-nested-list__icon sap-icon--home"></span>
                     <span class="fd-nested-list__title">Level 1 Item</span>
                 </a>
             </li>
             <li class="fd-nested-list__item">
                 <a class="fd-nested-list__link is-selected" href="#">
-                    <span class="fd-nested-list__icon sap-icon--calendar"></span>
+                    <span aria-hidden="true" class="fd-nested-list__icon sap-icon--calendar"></span>
                     <span class="fd-nested-list__title">Level 1 Item</span>
                 </a>
             </li>
             <li class="fd-nested-list__item">
-                <div class="fd-nested-list__content has-child" tabindex="0">
-                    <a class="fd-nested-list__link" href="#" tabindex="-1">
-                        <span class="fd-nested-list__icon sap-icon--employee"></span>
+                <div class="fd-nested-list__content has-child">
+                    <span class="fd-nested-list__link">
+                        <span aria-hidden="true" class="fd-nested-list__icon sap-icon--employee"></span>
                         <span class="fd-nested-list__title">Level 1 Item</span>
-                    </a>
-                    <a class="fd-nested-list__expand-icon" href="#" tabindex="-1" aria-controls="EX500L2" aria-haspopup="true"></a>
+                    </span>
+                    <button class="fd-button fd-nested-list__button" aria-controls="EX100L2" aria-haspopup="true" aria-expanded="false"></button>
                 </div>
```

Comment:
It has to be a button as it doesn't navigate anywhere, it triggers the submenu. For accessibility purposes, the only things that should be anchors are links that perform navigation.
Do you mean that the element that triggers the submenu could also be a link, and the submenu can be triggered separately? If so I'll revert the first span to a link.

Reply:
Hi @meganaconley, element that triggers submenu can also be a link, there is separation between those 2 elements. The rest is ok.
![sidenav](https://user-images.githubusercontent.com/26483208/82318770-b492fa80-99d0-11ea-954b-502bdcfebcc2.gif)

Reply:
Regarding removing `fd-button` class, I was under the impression we established a pattern that we are _not_ using novel buttons and are applying style overrides to the base `fd-button` where necessary. If that's changed, it's fine, but we really need to be consistent.

Reply:
From the attached gif, it looks like 'Item 2' is acting as button, not a link.

Reply:
I've updated the nested list header to be a link as well. It does seem odd as the hover style applies to the entire line but now there are two separate clickable elements where as the gif demonstrates previously the entire row acted as a button.

Reply:
> From the attached gif, it looks like 'Item 2' is acting as button, not a link.

That's the problem of anchor because it can also look like a button

Reply:
Actually looking at this again, does this mean that the condensed state can't have a submenu? Because there is no room for a button, but the header is now a link:

<img width="72" alt="Screen Shot 2020-05-22 at 9 36 52 AM" src="https://user-images.githubusercontent.com/57502177/82689859-feb5ef00-9c0f-11ea-842b-d9a19613ffff.png">

Reply:
Condensed state appears to have submenus as popovers in this UI5 example: https://sapui5.hana.ondemand.com/#/entity/sap.tnt.SideNavigation/sample/sap.tnt.sample.SideNavigation

Reply:
Ok, it does look like in the collapsed state they only behave as buttons, I'll update the markup for that.